### PR TITLE
Bug 1228554 - Release gaiatest-v2.5 to PyPI

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1f85bbd0ba9330be75420c9b2fc5610d9fac90f7

File: tests/python/gaia-ui-tests/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-boto>=2.32.1
+boto==2.38.0
 marionette_driver==1.0.0
 marionette_client==1.0.0
-requests
+requests==2.8.1
 treeherder-client==1.5
```
Comment (askeing) [inline tests/python/gaia-ui-tests/requirements.txt]: the treeherder-client is 1.8.0 now

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c59b923c70e200455327a37ad585f7fbaf17d67

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9029172bbbe324674c3b61346340a0e44a4d05a0

**Pull request merged**